src/coord/rmr/rq.c

```diff
@@ -181,21 +181,19 @@ static struct queueItem *rqPop(MRWorkQueue *q) {
 
     // Handle pending info logging. Access only to a non-NULL head and pendingInfo,
     // So it's safe to do without the lock.
-    const char *logLevel = "verbose";
-    if (q->head == q->pendingInfo.head) {
-      // If we hit the same head multiple times, we may have a problem. Increase log level.
-      if (++q->pendingInfo.hitCount > 100) logLevel = "notice";
+    if (q->head == q->pendingInfo.head && q->sz > q->pendingInfo.warnSize) {
```

Comment:
Coverage?

Reply:
This is only possible under heavy loads, which we don't have a way to replicate in our test suit today